core/services/chainlink/relayer_factory.go

```diff
@@ -72,6 +74,7 @@ func (r *RelayerFactory) NewEVM(ctx context.Context, config EVMFactoryConfig) (m
 
 		relayerOpts := evmrelay.RelayerOpts{
 			DS:                   ccOpts.DS,
+			DBURL:                &dburl,
```

Comment:
Cool, so... we tell all relays to use the databse url in the global toml config (`config.AppConfig.Database().URL()`) by default, but allow it to be overridden on a per-relay basis by reading the `CL_DATABASE` environment variable that's set locally on each running relay? I think that makes sense